### ISSUE TITLE
[BUG]  ERROR: list index out of range

### ISSUE THREAD

**Issue by AndyLavr** [aristocratos/bpytop#5]
```bash
03/08/20 (17:31:03) | ERROR: list index out of range
Traceback (most recent call last):
  File "./bpytop.py", line 2442, in _collect_temps
    cls.cpu_temp[THREADS // 2 + n].append(t)
IndexError: list index out of range
```

Fix:

```bash
--- bpytop.py.orig      2020-08-03 18:17:37.982721770 +0300
+++ bpytop.py   2020-08-03 18:16:07.183272634 +0300
@@ -2437,7 +2437,7 @@
                                if len(cores) < THREADS:
                                        if cpu_type == "intel" or (cpu_type == "other" and len(cores) == THREADS // 2):
                                                cls.cpu_temp[0].append(temp)
-                                               for n, t in enumerate(cores, start=1):
+                                               for n, t in enumerate(cores, start=0):
                                                        cls.cpu_temp[n].append(t)
                                                        cls.cpu_temp[THREADS // 2 + n].append(t)
                                        elif cpu_type == "ryzen" or cpu_type == "other":
```

**Comment by aristocratos** [comment]
That would overwrite any values fetched from "Package" or "Tdie" with .

I rather go with an IndexError catch, since any value out the Index range would never be used anyways.

Would you mind running `python3 -c "import psutil; print(psutil.sensors_temperatures())"`

Not quite sure why you are getting uneven number of core temps.

**Comment by aristocratos** [comment]
Fixed in v1.0.2

**Comment by AndyLavr** [comment]
> python3 -c "import psutil; print(psutil.sensors_temperatures())"

```
$ python3 -c "import psutil; print(psutil.sensors_temperatures())"
{'acpitz': [shwtemp(label='', current=25.0, high=107.0, critical=107.0)], 'amdgpu': [shwtemp(label='edge', current=52.0, high=104000.0, critical=104000.0)], 'coretemp': [shwtemp(label='Package id 0', current=63.0, high=86.0, critical=100.0), shwtemp(label='Core 0', current=62.0, high=86.0, critical=100.0), shwtemp(label='Core 1', current=63.0, high=86.0, critical=100.0), shwtemp(label='Core 2', current=56.0, high=86.0, critical=100.0), shwtemp(label='Core 3', current=55.0, high=86.0, critical=100.0)], 'dell_smm': [shwtemp(label='CPU', current=55.0, high=None, critical=None), shwtemp(label='Ambient', current=42.0, high=None, critical=None), shwtemp(label='SODIMM', current=36.0, high=None, critical=None), shwtemp(label='SODIMM', current=40.0, high=None, critical=None), shwtemp(label='SODIMM', current=37.0, high=None, critical=None), shwtemp(label='GPU', current=50.0, high=None, critical=None)]}
```

**Comment by AndyLavr** [comment]
> Fixed in v1.0.2

Thanks! Version 1.0.2 work fine!
